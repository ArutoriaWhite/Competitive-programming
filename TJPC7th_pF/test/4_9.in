1527774 965
1751 0
3170 0
7865 0
8885 1
9507 1
11145 0
11843 0
13355 1
17103 1
17140 1
19342 0
19446 0
21020 0
21758 0
22394 1
22595 0
22905 0
23893 0
26095 1
26146 0
27689 1
31179 0
31930 0
32323 1
33220 0
35684 0
36248 0
37255 1
38010 1
38174 0
41394 1
44288 0
45980 1
45997 0
47513 0
48038 1
48377 0
48652 1
49527 0
49655 1
49880 0
51957 0
55465 1
55744 0
56595 1
60368 0
60470 0
67359 1
68344 1
69712 1
71807 0
72157 1
74332 0
74420 0
74462 0
74632 1
78514 1
78542 0
83206 0
84351 1
86656 1
88077 1
89037 0
90549 1
91034 1
95609 0
96047 1
96883 1
96961 1
101900 1
102314 0
107116 1
110388 1
111692 1
112531 1
113425 0
116961 0
117602 0
117981 1
118256 1
119646 1
119655 1
120645 1
120840 1
123908 0
124059 1
124884 1
126042 1
126756 1
128721 0
130860 1
131754 0
131789 1
133068 1
133803 0
136125 1
136414 1
136550 0
142603 1
142789 1
142838 1
144755 0
146168 1
148086 1
148653 0
150425 0
152247 0
154332 0
157738 1
159229 1
161037 1
162128 0
163476 0
163549 0
166021 0
167785 0
170141 1
171331 1
171617 1
171860 0
171868 1
171910 0
172133 1
172995 0
173016 1
173196 1
174830 0
176892 0
178454 0
179447 1
181009 1
183884 1
185812 1
185988 1
187367 0
189790 0
193050 0
195565 0
196111 0
198059 0
198547 0
201909 1
209251 0
209705 0
209867 0
211284 1
216310 0
217438 0
218182 0
219200 0
219380 1
222328 1
224903 0
224994 1
225997 0
226190 0
226899 1
227460 0
228517 1
231978 0
236161 0
236374 0
236907 0
237507 1
239021 1
240232 1
241411 1
242864 0
244102 0
245513 1
245942 1
246732 1
246757 0
253767 0
256474 0
256766 0
257006 1
259555 1
259670 1
260160 0
260406 0
263670 0
265191 0
266027 1
266309 1
269036 0
270139 1
271543 1
272557 0
273829 1
276598 1
278563 1
280889 1
281556 1
283173 1
285127 0
285558 1
288830 1
288842 0
288971 1
290843 1
291435 1
292681 1
293194 1
293355 1
294493 1
296579 1
298110 0
300073 1
302426 1
310154 0
310883 1
312312 1
315156 0
315469 1
316109 0
317006 1
318685 0
320729 0
322202 0
322283 1
324578 1
325298 1
326320 1
328383 0
330436 1
330966 0
331124 0
332471 0
334138 0
334229 1
337187 0
339953 1
339988 0
344331 1
344398 1
349962 1
351629 0
352453 0
352815 0
353557 1
353623 0
354832 1
356432 1
361400 1
362857 0
363154 0
363750 0
364595 1
364911 1
366806 1
368128 1
369381 1
371187 0
373113 0
375251 0
378530 0
379093 0
381589 0
383043 0
383107 1
384930 1
385122 0
388013 0
389378 1
390689 1
392963 0
393930 0
395339 1
397945 1
398188 0
398255 1
398923 1
400844 0
401522 0
401780 1
402069 1
404332 0
407154 1
408121 0
414725 1
415325 0
416778 1
417278 0
420783 1
429048 1
429717 0
431333 1
436570 0
437525 1
438480 1
438872 1
441221 1
442609 0
442980 0
446171 1
448565 1
450228 1
450584 0
452539 1
453869 0
454299 0
457377 0
458893 1
459006 1
461802 0
463102 1
480383 0
481637 0
482067 1
483252 1
483999 1
485666 0
485877 1
486119 0
487717 0
490799 0
491327 1
492632 1
495025 0
498644 0
498777 0
498896 1
502783 1
503688 1
503949 0
505212 0
506506 0
506533 1
507532 0
519952 1
522881 1
525594 1
527230 0
529077 1
535236 0
536416 0
537172 0
540434 0
543043 0
546249 0
546597 0
546608 1
546843 0
547764 0
550020 1
550104 1
550534 0
551608 1
554367 0
554896 1
555200 0
555943 1
559564 1
560768 0
562135 1
562288 0
566169 1
566645 0
567490 1
568248 0
568817 0
571628 1
572626 1
573215 1
573859 0
574263 1
575601 1
576709 0
578535 1
578556 0
579068 1
580528 1
581091 1
583514 0
584449 1
585332 1
586263 1
588018 1
590332 0
590781 0
595459 1
596243 0
597065 0
597246 1
599064 1
600223 1
602383 1
602448 1
603019 0
605611 0
611726 1
612190 1
612692 1
612990 1
613214 1
615219 0
615377 1
615723 0
617269 0
618458 0
619537 1
619947 1
620904 1
621203 1
621210 0
625038 1
629235 1
629332 0
629803 1
630185 0
630584 1
631254 0
632936 0
633029 1
636078 0
636156 1
636826 1
637223 0
640228 0
643125 1
643619 0
647433 0
650804 0
651171 0
652366 0
652914 1
653374 0
653586 0
654685 1
657099 0
659555 1
661475 0
663809 0
665399 0
668093 1
669817 0
670596 1
672125 1
674663 1
674882 0
675376 0
676447 1
676892 0
680533 1
683992 0
685993 1
688515 1
688924 0
691994 1
693904 0
693987 0
697122 1
700931 0
701184 0
702080 0
703176 0
704018 1
706018 0
707362 1
708205 0
710873 0
711373 0
713865 0
715630 1
718173 1
721534 1
721617 1
721760 0
724221 1
724405 1
728987 0
730479 1
730494 1
733675 1
733724 1
735865 0
737703 1
739190 1
739250 0
744897 1
748860 0
752949 0
754054 0
755008 0
756101 1
759238 0
761183 0
762922 0
763640 1
763786 0
763984 0
767419 1
769612 1
774905 0
778200 0
781286 0
783969 0
784967 0
785588 0
787236 1
792161 1
792984 1
793889 0
794940 1
796119 1
800354 0
800919 1
801785 0
802349 0
803144 1
803897 1
807355 1
808523 1
810138 0
810589 0
810790 1
810979 1
811069 0
811090 1
814062 0
815649 0
817253 0
817327 1
818426 0
818915 1
819122 1
819714 0
819785 0
821082 0
828794 1
828894 1
830462 1
830589 1
831902 1
834046 1
834359 0
834625 1
834750 1
836215 1
845432 1
845442 0
845563 1
847532 0
848504 1
849007 0
853660 1
853970 1
856421 0
859136 1
860022 1
862784 1
863759 1
864881 0
867168 0
871037 0
873969 0
879447 1
882954 1
883986 1
884480 0
884881 0
884895 1
887248 1
887273 1
890260 1
891818 1
891913 0
892460 1
893379 0
895849 1
896461 0
896504 1
902772 1
902804 1
903608 1
904424 0
907820 1
914846 0
916284 0
916746 0
917905 0
918557 1
918926 1
921505 1
922872 1
924075 0
924259 1
928726 0
929172 0
931327 1
932979 1
936600 0
936734 0
938482 0
940687 0
941956 1
944586 1
945729 0
946150 0
946272 0
946680 1
949515 1
950105 0
951413 0
951541 0
951740 1
952418 0
954151 0
955028 1
956872 0
957982 0
958252 1
962890 0
962948 1
963558 1
964335 0
966183 1
966351 0
967590 0
970223 0
973867 1
973982 1
974754 1
974830 1
977433 1
978038 1
979235 1
983256 1
987670 1
990224 0
990475 1
996275 0
998321 1
999530 0
1000531 0
1002861 0
1004209 0
1004727 1
1006340 0
1008326 0
1009349 1
1013511 0
1014924 0
1019196 0
1019508 1
1020963 0
1021272 0
1022342 0
1023080 0
1025260 0
1025550 0
1025769 0
1026299 1
1026700 1
1027456 1
1028248 1
1028711 0
1028895 1
1030353 0
1030395 0
1030794 0
1030840 0
1031892 0
1032959 0
1034360 1
1035909 0
1036769 0
1038949 0
1039857 0
1040296 0
1046815 1
1048658 1
1049267 1
1056395 0
1059298 0
1060927 0
1062658 1
1063010 1
1063943 1
1068127 0
1071856 0
1074467 1
1075383 1
1076458 0
1078006 1
1078335 1
1083584 0
1085499 1
1086792 0
1088137 1
1090157 1
1091844 1
1092030 0
1092516 1
1096371 0
1096463 0
1097483 0
1098747 1
1099148 0
1100325 0
1104028 1
1105255 0
1109763 1
1113353 1
1114800 0
1119515 0
1119818 1
1121091 0
1121803 1
1123124 0
1123281 0
1124199 0
1125633 0
1125664 1
1134439 1
1138199 0
1138664 1
1139091 0
1140249 0
1140573 1
1140924 0
1141260 1
1144202 0
1145525 0
1155760 0
1156147 1
1157724 0
1158990 0
1161491 1
1164055 1
1164794 0
1169856 1
1172585 0
1172865 1
1173836 0
1176942 0
1177197 1
1178212 1
1178602 0
1178734 1
1181461 0
1181885 1
1187365 1
1189294 0
1191578 1
1191755 1
1191780 0
1191836 0
1192353 0
1193133 0
1195710 0
1196041 1
1199771 0
1200342 0
1201071 1
1203376 1
1203541 1
1203977 1
1204893 1
1205048 0
1207970 1
1210340 1
1211991 0
1214979 0
1215393 0
1218014 1
1220309 1
1223557 1
1226751 0
1228086 1
1229197 1
1231298 0
1231645 1
1234527 1
1236717 1
1237890 0
1239638 0
1243025 1
1244866 1
1246065 1
1246409 1
1248182 1
1249808 0
1250191 0
1251081 0
1251554 1
1253686 0
1254329 1
1254698 0
1256392 1
1256556 0
1257432 0
1257503 0
1259718 1
1259981 1
1263126 0
1263543 1
1263990 0
1264615 0
1265954 1
1266797 1
1266889 1
1267999 1
1268423 1
1269375 0
1271706 1
1272492 1
1274237 0
1277503 0
1278250 0
1278461 1
1278488 0
1279154 0
1279196 1
1279493 1
1281477 1
1282284 1
1285520 0
1286371 0
1287484 1
1288050 0
1289022 0
1290909 0
1291768 0
1292728 0
1292878 1
1292990 1
1294461 0
1300395 0
1306548 0
1308067 0
1308297 1
1309502 0
1312684 0
1314543 1
1314736 1
1316597 1
1316815 0
1327785 1
1329214 0
1329783 1
1330328 1
1332034 0
1332094 0
1332363 1
1333414 0
1334526 0
1335235 0
1342273 1
1343603 1
1346118 1
1347388 0
1351044 0
1354415 0
1355031 1
1356061 1
1357352 1
1359066 0
1365889 0
1369344 0
1370897 0
1371091 1
1371242 1
1374410 0
1379195 0
1381196 0
1382333 0
1382827 0
1383123 0
1383762 1
1386442 1
1388418 0
1389730 0
1389983 0
1393878 1
1395362 0
1397997 0
1399730 0
1399992 1
1401105 1
1401599 0
1403835 1
1404566 0
1405736 0
1406720 1
1409060 0
1410167 0
1411966 0
1417823 1
1418004 0
1418189 1
1418282 0
1418461 0
1420261 1
1422252 1
1424311 1
1426747 1
1428117 0
1428673 0
1432524 1
1432775 1
1433860 1
1435667 1
1437539 0
1439971 0
1440197 0
1441902 1
1443884 0
1444350 0
1445052 0
1445113 0
1449264 1
1450006 0
1451698 0
1452853 0
1454929 1
1455336 0
1456360 0
1459780 0
1461822 1
1462240 0
1465281 1
1465354 1
1471489 0
1471555 0
1472257 0
1473000 1
1473725 1
1475122 1
1475139 0
1475950 0
1477522 0
1478121 0
1479149 1
1480674 1
1480743 1
1481023 0
1482006 1
1482572 1
1482629 1
1483986 0
1487480 0
1487900 0
1489644 1
1491761 0
1493096 0
1493660 0
1495832 0
1498218 0
1498803 0
1499087 1
1499328 0
1502959 0
1506178 1
1507968 1
1510390 0
1512427 1
1512670 0
1516571 1
1517135 0
1518955 0
1522166 1
1526246 1
1526420 1
