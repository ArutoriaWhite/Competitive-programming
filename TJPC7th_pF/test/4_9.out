198547 1517070
